pennylane_qiskit/converter.py

```diff
@@ -224,16 +225,43 @@ def _function(params: dict = None, wires: list = None):
                 gate = dagger_map[instruction_name]
                 qml.adjoint(gate)(wires=operation_wires)
 
+            elif isinstance(op, Measure):
+                meas_terminal = True
+                # Look-ahead for more gate(s) on its wire(s)
+                for meas_ix in range(idx, num_inst - 1):
+                    (next_op, next_qargs, _), op_wires = qc.data[meas_ix + 1], set(operation_wires)
+                    next_op_name = getattr(next_op, "base_class", next_op.__class__).__name__
+                    if next_op_name not in ["Barrier", "GlobalPhaseGate"]:
+                        next_op_wires = set(wire_map[hash(qubit)] for qubit in next_qargs)
+                        if next_op_wires.intersection(op_wires):
```

Comment:
```suggestion
                        if next_op_wires.intersection(set(operation_wires)):
```
I was momentarily confused trying to figure out how the contents of `op_wires` differ from `operation_wires`. Seems cleaner to just convert it to a set here, rather than making a new variable to represent the same information in set form. Then the `op_wires` definition can be removed from 232.

Reply:
Looks like I misplaced the `op_wires` last time. I wanted to keep it on the top so that I don't recompute it every time for hitting the `if` condition.